src/audio/base_fw.c

```diff
@@ -497,6 +520,7 @@ static int basefw_get_large_config(struct comp_dev *dev,
 	case IPC4_DSP_RESOURCE_STATE:
 	case IPC4_NOTIFICATION_MASK:
 	case IPC4_MODULES_INFO_GET:
+		return  basefw_modules_info_get(data_offset, data);
```

Comment:
this handler will also handle `IPC4_DSP_RESOURCE_STATE` and `IPC4_NOTIFICATION_MASK`?